redwood-composeui/src/commonMain/kotlin/app/cash/redwood/composeui/RedwoodContent.kt

```diff
@@ -56,25 +52,23 @@ public fun RedwoodContent(
     viewportSize = viewportSize,
     density = density.density.toDouble(),
   )
-
-  val redwoodView = remember {
-    object : RedwoodView<@Composable () -> Unit> {
-      override val children = ComposeWidgetChildren()
-      override val onBackPressedDispatcher = onBackPressedDispatcher
-      override val uiConfiguration = MutableStateFlow(uiConfiguration)
-      override fun reset() {
-        children.remove(0, children.widgets.size)
-      }
-    }
-  }
-  LaunchedEffect(redwoodView, uiConfiguration) {
-    redwoodView.uiConfiguration.value = uiConfiguration
+  val uiConfigurations = remember { MutableStateFlow(uiConfiguration) }
+  LaunchedEffect(uiConfiguration) {
+    uiConfigurations.value = uiConfiguration
```

Comment:
What does this do?

Reply:
Sends the new value into the `StateFlow` (if it has changed) so it can be observed by the Redwood compose code